config/locales/en.yml

```diff
@@ -273,9 +273,19 @@ en:
   static_pages:
     not_authorised:
       title: You are not yet authorised to use the Teaching Jobs service
-      text:
-        - You can register your interest to use the service by emailing <a href='mailto:teachingjobs@digital.education.gov.uk', class='govuk-link'>teachingjobs@digital.education.gov.uk</a>. It is currently open to schools in Cambridgeshire and the North East but will be rolled out in phases to other areas across England.
+      without_email:
+        text:
+          - You can register your interest to use the service by emailing <a href='mailto:teachingjobs@digital.education.gov.uk', class='govuk-link'>teachingjobs@digital.education.gov.uk</a>. It is currently open to schools in Cambridgeshire and the North East but will be rolled out in phases to other areas across England.
 
-        - If your Teaching Jobs account has already been set up but you are unable to sign in, email the same address with “help” in the subject line.
+          - If your Teaching Jobs account has already been set up but you are unable to sign in, email the same address with "help" in the subject line.
+      with_email:
+        text:
+          - You've tried to sign in with %{email}, which is not associated with a Teaching Jobs account. If you have an account associated with another email, please first sign %{email} out of DfE Sign-in. Then sign into Teaching Jobs at <a href='https://profile.signin.education.gov.uk/' class='govuk-link'>https://profile.signin.education.gov.uk/</a> using your authorised email.
```

Comment:
Can the link be extracted to a locale variable so we can configure it from the view please.